lib/cookbook-omnifetch/artifactserver.rb

```diff
@@ -43,10 +43,13 @@ def install
       http = http_client(uri)
       http.streaming_request(nil) do |tempfile|
         tempfile.close
+        # TODO: What are the ACLs supposed to be?  Windows might or might not
```

Comment:
Is there any way to resolve this question now? I suspect everything should be fine. The tarball will probably have very limited permissions, but then we open it and create new files in the user's home dir which should get us sane permissions.